### PR TITLE
修复获取收集器状态卡住的问题

### PR DESCRIPTION
**Pull request by redHJ**
@cyzhou1996 
@liukai2012

### CODE REVIEW

File: mgr/runner.go [hunk 1]
```diff
@@ -716,7 +716,6 @@ func (r *LogExportRunner) rawReadLines(dataSourceTag string) (lines, froms []str
 		} else {
 			r.rs.ReaderStats.LastError = TruncateStrSize(err.Error(), DefaultTruncateMaxSize)
 		}
-		time.Sleep(r.backoff.Duration())
```
Comment (redHJ) [inline mgr/runner.go:719]: https://github.com/qiniu/logkit/pull/1077/files#diff-3332a0a32b5997195d5eec04225db1c5R680 和
https://github.com/qiniu/logkit/pull/1077/files#diff-3332a0a32b5997195d5eec04225db1c5R685
已经sleep过了

**Comment by liukai2012** [comment]
lgtm

**Comment by cyzhou1996** [comment]
lgtm

File: mgr/runner.go [hunk 2]
```diff
@@ -725,9 +724,9 @@ func (r *LogExportRunner) rawReadLines(dataSourceTag string) (lines, froms []str
 		r.historyMutex.Unlock()
 	} else {
 		r.rs.ReaderStats.LastError = ""
-		r.backoff.Reset()
```
Comment (redHJ) [inline mgr/runner.go:728]: https://github.com/qiniu/logkit/pull/1077/files#diff-3332a0a32b5997195d5eec04225db1c5R688 已有

**Pull request merged**